gravitee-apim-console-webui/src/management/api/api-traffic-v4/webhook-logs-details/webhook-logs-details.component.scss

```
@@ -0,0 +1,334 @@
+@use 'sass:map';
+@use '@angular/material' as mat;
+@use '@gravitee/ui-particles-angular' as gio;
+@use 'src/scss/gio-layout' as gio-layout;
+
+$typography: map.get(gio.$mat-theme, typography);
+
+:host {
+  @include gio-layout.gio-responsive-margin-container;
+}
+
+.card {
+  &__header {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 24px;
+
+    &__title {
+      @include mat.m2-typography-level($typography, headline-6);
+      margin: 0;
+      font-size: 18px;
+      line-height: 24px;
+      font-weight: bold;
+    }
+  }
+}
+
+.card__container {
+  display: flex;
+  flex-direction: column;
+  gap: 24px;
+}
+
+.section {
+  border-radius: 16px;
+  padding: 24px;
+  display: flex;
+  flex-direction: column;
+  gap: 20px;
+
+  &--details {
+    background: mat.m2-get-color-from-palette(gio.$mat-space-palette, 'lighter96');
```

I don't think 96 exists 🤔 
```suggestion
    background: mat.m2-get-color-from-palette(gio.$mat-space-palette, 'lighter90');
```

---

Def not. Updated thanks for catching :)